src/checked_transaction.rs

```diff
@@ -467,7 +467,7 @@ mod tests {
         // verify available balance was decreased by max fee
         assert_eq!(
             checked.initial_free_balances[&AssetId::default()],
-            input_amount - checked.max_fee
```

Comment:
Why in some places function call, in some not?=)

Reply:
I used pub fn's when they were available, but AvailableBalances is a private type and doesn't have any accessor fns.